mlos_core/mlos_core/tests/optimizers/optimizer_test.py

```diff
@@ -192,7 +189,13 @@ def objective(point: pd.DataFrame) -> pd.Series:   # pylint: disable=invalid-nam
         space_adapter_type=SpaceAdapterType.LLAMATUNE,
         space_adapter_kwargs=space_adapter_kwargs
     )
+
+    # Initialize an optimizer that uses the original space
+    optimizer: BaseOptimizer = OptimizerFactory.create(input_space, optimizer_type, optimizer_kwargs=kwargs)
```

Comment:
We'll need named parameters here. Please merge the latest master, resolve the conflicts, and fix that part
```suggestion
    optimizer: BaseOptimizer = OptimizerFactory.create(
        parameter_space=input_space,
        optimizer_type=optimizer_type,
        optimizer_kwargs=kwargs,
    )
```